language-configuration.json

```diff
@@ -22,5 +22,6 @@
     ["[", "]"],
     ["(", ")"],
     ["\"", "\""]
-  ]
+  ],
+  "wordPattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\#\\%\\^\\&\\*\\(\\)\\-\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s]+)"
```

Comment:
what a beautiful pattern 💩

Reply:
I used the word pattern from here: https://code.visualstudio.com/api/language-extensions/language-configuration-guide#word-pattern and removed `@`